tests/common-tests.h

```diff
@@ -4171,6 +4171,7 @@ TEST_CASE_METHOD(common_tests, "Blank padding", "[core][insert][exception]")
     CHECK(tvarchar == test1);
 }
 
+
```

Comment:
@vadz No. At the first step, I've added test case here.
But, unfortunately, connection_parameters coupled with the backend, which fails common part tests other than MySQL. So I moved it to the specific test file.